knip.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import type { KnipConfig } from 'knip'
+
+const config: KnipConfig = {
```

Comment:
Can you supply these options for the `knip` command inline so we wouldn't need to add a new file to the root of the project? I'm trying to avoid that as for newcomers these files can feel a bit daunting